src/components/experiments/ExperimentList.jsx

```diff
@@ -1,29 +1,56 @@
 import React from "react";
 import { connect } from "react-redux";
 import { fetchExperiments } from "../../state/experiments/actions";
-import { getExperimentsCount } from "../../state/experiments/selectors";
+import {
+  getExperimentsCount,
+  getFilteredExperimentsByDate
+} from "../../state/experiments/selectors";
+import { getStartDate, getEndDate } from "../../state/dates/selectors";
 import PropTypes from "prop-types";
 
 export class ExperimentList extends React.Component {
   componentDidMount() {
     this.props.fetchExperiments();
   }
 
+  renderTitle() {
+    let title = "experiments";
+    if (this.props.startDate && this.props.endDate) {
```

Comment:
This is still the same polynomial strategy we talked about yesterday.  How will this look when we add a ton more filters?

Reply:
But also very nice putting it in a nice little method 👍